src/script/common/c_content.h

```diff
@@ -77,130 +77,99 @@ extern struct EnumString es_TileAnimationType[];
 
 extern const std::array<const char *, 33> object_property_keys;
 
-void               read_content_features     (lua_State *L, ContentFeatures &f,
-                                              int index);
-void               push_content_features     (lua_State *L,
-                                              const ContentFeatures &c);
+void read_content_features (lua_State *L, ContentFeatures &f, int index);
```

Comment:
```suggestion
void read_content_features(lua_State *L, ContentFeatures &f, int index);
```
The space was just there for the alignment. If we remove the alignment (which is fine imo), don't keep the one space.
(Also everywhere below.)